libs/common/src/types/mongo-db-models/deals.ts

```diff
@@ -8,9 +8,11 @@ type BssEwcsDeal = AnyObject & {
   };
 };
 
-type GefDeal = AnyObject & {
+export type GefDeal = AnyObject & {
   dealType: typeof DEAL_TYPE.GEF;
-  ukefDealId: string;
+  ukefDealId: string | null;
+  eligibility: AnyObject;
+  exporter: AnyObject;
```

Comment:
Good job, this slow build out of these models as we understand them more is exactly what we need